tests/builder/default_vals.rs

```diff
@@ -315,6 +315,26 @@ fn default_if_arg_present_with_value_no_default() {
     );
 }
 
+#[test]
+fn default_values_if_arg_present_with_value_no_default() {
```

Comment:
If the tests aren't being added before the commit, could you add them in the same commit?